keyboards/bm16s/keymaps/default/keymap.c

```diff
@@ -0,0 +1,20 @@
+#include "bm16s.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```

Reply:
I looked into this and couldn't get it to compile, there might be something I'm missing.

Reply:
Make sure your repo is up to date.   The `QMK_KEYBOARD_H` is automatically generated based on the keyboard.... so it should "just work". 

Otherwise, what error do you see, specifically.

Reply:
Woke up this morning with more time to work on this and now it's working fine, no idea, it was something about not being able to find the import, but apparently not an issue anymore.